docs/content/en/docs/core-components/modules.md

```diff
@@ -32,7 +32,7 @@ In every other circumstance the step is considered to have failed.
 
 #### Variables
 
-This module does not define specific variables as input, but variable interpolation is supported in the command and target definitions. It has the following output variables:
+This module does not define specific variables as input, but variable interpolation is supported in the command and target definitions. It has `one` output variable of type string (see the `__soarca_ssh_result__`). If you want to use the output variable in your next step you will need to define a variable in your playbook as `step or playbook variable` that is of the same type. Also you need to specify an `out_args` key see `__your_step_output_variable__` in the example. Note you can only use `one out_arg key` for this capability.
```

Comment:
Two points:
1. We say that the output variable defined in the step can be used in the next steps only because in SOARCA we use global scope for any defined variable, right? We should mention this, as it's not exactly adhering to the spec.
2. Besides than being the same type, it should also have the exact same name, right? Perhaps we can say that as well